pulpcore/cli/rpm/repository.py

```diff
@@ -82,6 +82,12 @@ def _content_callback(ctx: click.Context, param: click.Parameter, value: Any) ->
 CONTENT_LIST_SCHEMA = s.Schema([{"pulp_href": str}])
 
 
+def choice_to_int_callback(ctx: click.Context, param: click.Parameter, value: Any) -> Optional[int]:
+    if value:
+        return int(value)
+    return None
+
```

Comment:
Thanks for adding the test!
I was sure there was a bug in here, because in python `0` evaluates to `False`. As your test demonstrates, this actually works. And i think i understood now that what actually is passed in here is `"0"`, a non empty string that  evaluates to `True` instead. I would have expected that we needed `if value is None` instead, and i'm still on the fence. But I think we can merge it that way.